examples/Firebase_ESP8266_Neopixel/colors_ext.h

```diff
@@ -0,0 +1,93 @@
+// The following methods are extracted from the Adafruit Neopixel example, strandtest.
+// https://github.com/adafruit/Adafruit_NeoPixel
+
+#ifndef COLORS_EXT_H
+#define COLORS_EXT_H
+
+// Input a value 0 to 255 to get a color value.
+// The colours are a transition r - g - b - back to r.
+uint32_t Wheel(Adafruit_NeoPixel* strip, byte WheelPos) {
+  WheelPos = 255 - WheelPos;
+  if(WheelPos < 85) {
+    return strip->Color(255 - WheelPos * 3, 0, WheelPos * 3);
+  }
+  if(WheelPos < 170) {
+    WheelPos -= 85;
+    return strip->Color(0, WheelPos * 3, 255 - WheelPos * 3);
+  }
+  WheelPos -= 170;
+  return strip->Color(WheelPos * 3, 255 - WheelPos * 3, 0);
+}
+
+
+
+// Fill the dots one after the other with a color
+void colorWipe(Adafruit_NeoPixel* strip, uint32_t c, uint8_t wait) {
```

Comment:
can you only keep colorWipe and move it to the main file, is that's the only one you're using?